### ISSUE TITLE
ckeditor_uploader outside of django admin

### ISSUE THREAD

**Issue by tisdall** [django-ckeditor/django-ckeditor#341]
Right now the `ckeditor_uploader` works fine outside of the django admin system.  However, the template currently loads "admin/css/base.css" from django admin which returns a 404 if django admin isn't installed.  The styling seems to be okay without it, so I'm wondering if it should either be removed or just the needed parts adapted into a new stylesheet included with `ckeditor_uploader`.

**Comment by tisdall** [comment]
Here's what it looks like without the admin/css/base.css:
![image](https://cloud.githubusercontent.com/assets/4029388/21891527/1311a772-d8a1-11e6-8e00-94a1f1e50192.png)
Here's what it looks like with it:
![image](https://cloud.githubusercontent.com/assets/4029388/21891642/add55e98-d8a1-11e6-982c-45b417e24736.png)

**Comment by tisdall** [comment]
here's just the parts of admin/css/base.css being used (excluding the use of the Roboto font which is just used on the "Embed Image" link):
```
/* GLOBAL DEFAULTS */

form {
    margin: 0;
    padding: 0;
}

/* FORM DEFAULTS */

input {
    margin: 2px 0;
    padding: 2px 3px;
    vertical-align: middle;
    font-family: "Lucida Grande", Verdana, Arial, sans-serif;
    font-weight: normal;
    font-size: 13px;
}

input[type=text] {
    border: 1px solid #ccc;
    border-radius: 4px;
    padding: 5px 6px;
    margin-top: 0;
}

input[type=text]:focus {
    border-color: #999;
}

/* FORM BUTTONS */

input[type=submit] {
    background: #79aec8;
    padding: 10px 15px;
    border: none;
    border-radius: 4px;
    color: #fff;
    cursor: pointer;
}

input[type=submit]:active,
input[type=submit]:focus,
input[type=submit]:hover {
    background: #609ab6;
}

input[type=submit].default {
    float: right;
    border: none;
    font-weight: 400;
    background: #417690;
}

input[type=submit].default:active,
input[type=submit].default:focus,
input[type=submit].default:hover {
    background: #205067;
}

/* MESSAGES & ERRORS */

ul.errorlist {
    margin: 0 0 4px;
    padding: 0;
    color: #ba2121;
    background: #fff;
}

ul.errorlist li {
    font-size: 13px;
    display: block;
    margin-bottom: 4px;
}

ul.errorlist li:first-child {
    margin-top: 0;
}

/* OBJECT HISTORY */

table#change-history {
    width: 100%;
}

table#change-history tbody th {
    width: 16em;
}

/* PAGE STRUCTURE */

#container {
    position: relative;
}
```

Here's what it looks like (with an error message to show that it's styled properly):
![image](https://cloud.githubusercontent.com/assets/4029388/21892667/e5ee8bca-d8a5-11e6-9d8f-a125ad5f8fec.png)

**Comment by tisdall** [comment]
I could provide a PR, but I'm not sure if it's better to just inline these styling into the one template it's used on, or make a separate .css file.  Any preferences?

**Comment by riklaunim** [comment]
it could be a separate css file as it's just for ckeditor part.

**Comment by tisdall** [comment]
issue resolved due to #342
